app/assets/javascripts/application/load-error-svg.js

```diff
@@ -1,8 +1,16 @@
 mumuki.load(function () {
-  [403, 404, 500].forEach(function (errorCode) {
-    var url = '/error_' + errorCode + '.svg';
-    $.get(url, function () {
-      console.log('Loaded', url);
-    });
+  var svgs = ['403', '404', '500', 'timeout_1', 'timeout_2', 'timeout_3'];
+
+  mumuki.errors = mumuki.errors || {};
+
+  svgs.forEach(function (svgErrorSuffix) {
+    var image = 'error_' + svgErrorSuffix;
+    var url = '/' + image + '.svg';
+    if (!mumuki.errors[image]) {
+      $.get(url, function (data) {
```

Comment:
We need to fix this. See https://github.com/mumuki/mumuki-laboratory/issues/909

Reply:
Ese arreglo va a estar en otra branch

Reply:
Is this still necessary?